projects/vertical-components/src/lib/button/text-button/text-button.component.scss

```diff
@@ -0,0 +1,56 @@
+:host {
```

Comment:
Duplicate style for all buttons? For example: border-radius is the same in every button